src/decorator.ts

```diff
@@ -85,7 +86,11 @@ export class Decorator {
                 this.unfoldedRanges.push(range)
                 continue
             }
-
+			if (classNames.length <= foldLengthThreshold) {
```

Comment:
Given that the settings indicate "the minimum number of classes required to fold", 'classNames.length" should ideally be less than, and not equal to the threshold.